src/SFA.DAS.ApplyService.Database/Tables/Contacts.sql

```diff
@@ -27,4 +27,11 @@ GO
 ALTER TABLE [dbo].[Contacts] CHECK CONSTRAINT [FK_Contacts_Organisations]
 GO
 
-CREATE UNIQUE INDEX [IXU_Contacts] ON [Contacts] ([Email])
\ No newline at end of file
+CREATE UNIQUE INDEX [IXU_Contacts] ON [Contacts] ([Email])
+GO
+CREATE INDEX [IX_Contacts_UKPRN] ON [Contacts] ([ApplyOrganisationId])
+GO
+CREATE INDEX [IX_Contacts_SignInId] ON [Contacts] ([SignInId])
+GO
+CREATE INDEX [IX_Contacts_SignInType] ON [Contacts] ([SignInType])

```

Comment:
I think this column is deprecated now (was used in the days of DfeSignIn and As Login)... although from the sounds of things to come DfeSignIn may come back